### ISSUE TITLE
More ways to create a New Topic

### ISSUE THREAD

**Issue by alexwebsites** [InvisionPowerServices/Mobile-App-Issues#42]
_Please use this issue only for specific feedback or ideas about particular features/screens. If you would like to share more general feedback about the app, please email mobilesupport@invisionpower.com_

_We'll respond to each piece of feedback, but may not address all feedback immedately. Feedback may be closed and moved to our private tracker for internal discussion._

## Platform

Which platform do you use?

- [x ] iOS
- [ ] Android

## What's your feedback?

I can see users going to the All Topics page as they do on the website for fluid view. Would be nice to add the create topic link on this page as well, although you would need some sort of drop down to pick the forum. Otherwise to create a topic link is only available if you go into a selected forum I believe.

**Comment by rikkitissier** [comment]
Yes, I agree. I haven't figured out how best to approach the node selector that'd be necessary yet (given the long list/sub-items/permissions stuff that'd be necessary) but I think this would make a lot of sense to have.
